### ISSUE TITLE
[22] Error message not associated with form field

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#295]
This issue was identified as part of the accessibility testing performed on the DTA website Jan 2017: https://drive.google.com/open?id=0B3SXZ9dr83v0Y2cwV21HdUZ1STQ

Applies: Email address field on Sign-up
Type: WCAG failure – 1.3.1 Info and Relationships
Platform: Desktop and mobile
Severity: Medium

The Email Address error message is not associated with the form field.
When a screen reader user submits the form with an invalid email address, focus moves to the Email Address field. A screen reader user hears that the field is invalid due to the aria-invalid="true" attribute, however the associated error message (e.g. “Please enter a valid email address”) is not announced.

![image](https://cloud.githubusercontent.com/assets/2324569/23931858/afeb6f2a-0988-11e7-80bb-a20002a07bab.png)
Image: Screen shot of Email Address field with error message ‘Please enter a valid email address’.

**Recommendation**

Use the aria-describedby attribute to link the error message with the relevant input field.
Add aria-describedby to the form field when there’s an error. The value of aria-describedby is an ID reference to the error message.

**Sample code**
```
<input type="email" ... id="mce-EMAIL" aria-required="true" aria-invalid="true" aria-describedby="mce-EMAIL-ERROR">
<div id="mce-EMAIL-ERROR" class="mce_inline_error">This field is required.</div>
```

**Comment by bjgordon** [comment]
Might be tricky as these errors are set by the mailchimp js. Try to add the aria-describedby tag as suggested. Report to mailchimp as well.
